opentelemetry-sdk/src/logs/simple_log_processor.rs

```diff
@@ -131,6 +132,20 @@ impl<T: LogExporter> LogProcessor for SimpleLogProcessor<T> {
             exporter.set_resource(resource);
         }
     }
+
+    #[cfg(feature = "spec_unstable_logs_enabled")]
+    fn event_enabled(
+        &self,
+        level: opentelemetry::logs::Severity,
+        target: &str,
+        name: Option<&str>,
+    ) -> bool {
+        if let Ok(exporter) = self.exporter.lock() {
+            exporter.event_enabled(level, target, name)
+        } else {
+            true
```

Comment:
Wondering why `true` is returned to enable events if the exporter cannot be locked?

Reply:
Good point! Not sure what is the right behavior here. 
Since `true` is the default, I opted for `true`. It can be argued that if the lock got poisoned, no logs can be exported anyway, so returning false also is valid.

Good to revisit this. I'll probably revisit the need of Mutex lock in SimpleLogProcessor, and use atomics to achieve the goal is ensuring one export() is active at a time. With that, we won't face the lock poison issue.